builtin/settingtypes.txt

```diff
@@ -864,6 +864,11 @@ liquid_update (Liquid update tick) float 1.0
 #    Stated in mapblocks (16 nodes)
 block_send_optimize_distance (block send optimize distance) int 4 2
 
+#    Optimize sending under ground blocks to the client.
```

Comment:
I think this needs to state this will occur 'beyond block send optimisation distance'.